docs/tfe-cert-rotation.md

```diff
@@ -15,42 +15,45 @@ Follow these steps to rotate the certificates for your TFE instance.
 
 1. Obtain your new TFE TLS certificate file and private key file, both in PEM format.
 
-2. Update the values of the existing secrets in AWS Secrets Manager (`tfe_tls_cert_secret_arn` and `tfe_tls_privkey_secret_arn`, respectively). If you need assistance base64-encoding the files into strings prior to updating the secrets, see the examples below:
+1. Update the values of the existing secrets in AWS Secrets Manager (`tfe_tls_cert_secret_arn` and `tfe_tls_privkey_secret_arn`, respectively). If you need assistance base64-encoding the files into strings prior to updating the secrets, see the examples below:
 
     On Linux (bash):
+
```

Comment:
Would it be helpful to put an example on an AWS CLI command update using that base64? (just as an idea):

```bash
aws secretsmanager put-secret-value \
  --secret-id sandbox-tfe-tls-cert-base64 \
  --secret-string "{\"tfe_cert.pem\": \"$(cat new_tfe_cert.pem | base64 -w 0)\"}"
```

Reply:
Please could you submit these suggestions as new PRs or issues so we can consider them?